prime-router/src/test/kotlin/fhirengine/translation/HL7toFhirTranslatorTests.kt

```diff
@@ -68,7 +68,7 @@ OBX|1|CWE|94558-4^SARS-CoV-2 (COVID-19) Ag [Presence] in Respiratory specimen by
 
         // Source: https://confluence.hl7.org/display/OO/v2+Sample+Messages
         val unsupportedHL7 = """
-MSH|^~\&#|NIST EHR|NIST EHR Facility|NIST Test Lab APP|NIST Lab Facility|20130211184101-0500||OML^O21^OML_O21|NIST-LOI_5.0_1.1-NG|T|2.5.1|||AL|AL|||||
+MSH|^~\&#|NIST EHR|NIST EHR Facility|NIST Test Lab APP|NIST Lab Facility|20130211184101-0500||OML^O33^OML_O33|NIST-LOI_5.0_1.1-NG|T|2.5.1|||AL|AL|||||
```

Comment:
Note: This is suppose to fail. Adding support for OML_O21 means I had to change this to an unsupported message type.

Reply:
Is this for fail test?

Reply:
@oslynn -- I believe so, yes. There's a test for a valid ORU message before this -- then this message is used as part of an assertFailure.